app/src/main/java/org/liberty/android/fantastischmemo/ui/RecyclerViewAdapter.java

```diff
@@ -32,22 +43,63 @@ public MultipleChoiceHolder onCreateViewHolder(ViewGroup parent, int viewType) {
     }
 
     @Override
-    public void onBindViewHolder(MultipleChoiceHolder multipleChoiceHolder, int position) {
+    public void onBindViewHolder(final MultipleChoiceHolder multipleChoiceHolder, final int position) {
+        final MultipleChoiceCard card = mcCards.get(position);
+
         multipleChoiceHolder.questionText.setText(mcCards.get(position).getQuestion());
-        multipleChoiceHolder.option1Text.setText("Option 1: " + mcCards.get(position).getOption1());
-        multipleChoiceHolder.option2Text.setText("Option 2: " + mcCards.get(position).getOption2());
-        multipleChoiceHolder.option3Text.setText("Option 3: " + mcCards.get(position).getOption3());
-        multipleChoiceHolder.option4Text.setText("Option 4: " + mcCards.get(position).getOption4());
-        multipleChoiceHolder.answerText.setText("Answer: " + mcCards.get(position).getAnswer());
+        multipleChoiceHolder.option1Text.setText("Option 1: " + card.getOption1());
+        multipleChoiceHolder.option2Text.setText("Option 2: " + card.getOption2());
+        multipleChoiceHolder.option3Text.setText("Option 3: " + card.getOption3());
+        multipleChoiceHolder.option4Text.setText("Option 4: " + card.getOption4());
+        multipleChoiceHolder.answerText.setText("Answer: " + card.getAnswer());
+
+        dbOpenHelper = AnyMemoDBOpenHelperManager.getHelper(mContext, dbPath);
+        multipleChoiceCardDao = dbOpenHelper.getMultipleChoiceDao();
+        multipleChoiceCardDao.setHelper(dbOpenHelper);
 
-        // Set onClick listener here
+        multipleChoiceHolder.cardView.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                showListItemPopup(view, card, position);
+            }
+        });
     }
 
     @Override
     public int getItemCount() {
         return mcCards.size();
     }
 
+    private void showListItemPopup(final View childView, final MultipleChoiceCard card, final int position) {
+        View view = childView.findViewById(R.id.rv_mc_question);
+        PopupMenu popup = new PopupMenu(mContext, view);
+        MenuInflater inflater = popup.getMenuInflater();
+        inflater.inflate(R.menu.mc_card_list_popup_menu, popup.getMenu());
+        popup.setOnMenuItemClickListener(new PopupMenu.OnMenuItemClickListener() {
+            @Override
+            public boolean onMenuItemClick(MenuItem menuItem) {
+                Intent myIntent = new Intent();
+                switch (menuItem.getItemId()) {
+                    case R.id.delete_mc:
+                        multipleChoiceCardDao.deleteMultipleChoiceCard(card);
+                        mcCards.remove(card);
+                        notifyItemRemoved(position);
```

Comment:
Why do you have to delete and then remove the card? Whats the difference?

Reply:
delete is removing it from the db. remove is removing it from the current static list. I need to do it for both so it updates the activity automatically without the need to refresh the page